assets/account/src/lib.rs

```diff
@@ -88,5 +101,35 @@ blueprint! {
                 }
             }
         }
+
+        /// Create proof of resource.
+        pub fn create_proof_by_amount(
+            &self,
+            amount: Decimal,
+            resource_def_id: ResourceDefId,
+        ) -> Proof {
+            let vault = self.vaults.get(&resource_def_id);
```

Comment:
Should we check for amount == 0 here? Should also add comment on what to expect if amount == 0

Reply:
Yes, I will make it clearer on what happens if `amount == zero`